CakePHP/Tests/Formatting/BlankLineBeforeReturnUnitTest.inc

```diff
@@ -91,3 +91,14 @@ function validFunctionReturnSix()
             return false;
     }
 }
+
+/**
+ * [validFunctionReturnSeven description]
+ *
+ * @return null
+ */
+function validFunctionReturnSeven()
+{
+    /** @var Foo $foo */
```

Comment:
Shouldnt we also fix that https://github.com/cakephp/cakephp-codesniffer/issues/172 issue that the inline doc blocks are fixed wrong in general? And that those are actually comments right now which they shouldnt be.
You can look into the existing sniffers I mentioned.

Reply:
That would require changing another sniff and beyond the scope of this PR.